Base/usr/share/man/man4/full.md

```diff
@@ -0,0 +1,35 @@
+## Name
+
+full - always full device
+
+## Description
+
+`/dev/full` is a character device which is always full.
+
+Reading from `/dev/full` returns '\0' bytes and exits with status 0.
+
+Writing to `/dev/full` fails with ENOSPC error.
+
+To create it manually:
+
+```sh
+mknod /dev/full c 1 7
+chmod 666 /dev/full
```

Comment:
Currently, for no adequately explained reason, `/dev/full` is created with `600` permissions by default (not `666`).

The intention of this device is to allow programs (including userland programs) to test error handling when writing to devices which are full. Limiting this to root only by default seems like an oversight.